worldedit-bukkit/adapters/adapter-1_19/src/main/java/com/sk89q/worldedit/bukkit/adapter/impl/fawe/v1_19_R1/PaperweightPlatformAdapter.java

```diff
@@ -106,6 +107,9 @@ public final class PaperweightPlatformAdapter extends NMSAdapter {
 
     private static final Field fieldRemove;
 
+    public static final boolean POST_CHUNK_REWRITE;
```

Comment:
```suggestion
   static final boolean POST_CHUNK_REWRITE;
```
We can safely weaken the access modifier.